src/app/logs-myplanet/logs-myplanet.component.ts

```diff
@@ -62,4 +62,13 @@ export class LogsMyPlanetComponent implements OnInit {
     }, (error) => this.planetMessageService.showAlert('There was a problem getting ' + this.childType));
   }
 
+  checkEmptyLogs() {
+    this.apklogs.forEach(element => {
```

Comment:
This doesn't return what you intend.  The `return false` is in the scope of the function parameter for `forEach` and does not affect `checkEmptyLogs`.  `checkEmptyLogs` always returns `true`.

Generally, you should not return a value in a `forEach` function since that always returns `undefined`.  Returning a value makes the code harder to read because at first glance it looks like it is doing something it is not.